vite.config.js

```diff
@@ -15,9 +23,9 @@ export default defineConfig({
   },
   build: {
     lib: {
-      entry: path.resolve(__dirname, 'src/index.js'),
+      entry: path.resolve(__dirname, 'src/index.ts'),
       name: 'Swimm UI',
-      fileName: (format) => `swimm-ui.${format}.js`,
+      fileName: (format) => `swimm-ui.${format}.ts`,
```

Comment:
That's not right. The output is supposed to be a JS file. Consider if we really a UMD build, and set `formats` accodrdingly.